adapter/PreciceInterface.h

```diff
@@ -101,11 +101,11 @@ typedef struct SimulationData {
 	char * set;
 	double * co;
 	ITG nboun;
-        ITG nforc; //total number of forces
+    ITG nforc; //total number of forces
```

Comment:
White spaces seem to have been mixed up a little. Please correct this (at several positions in the code).